runner.py

```diff
@@ -279,6 +279,9 @@ def runner(input_dir, reference_file, output_dir, max_basecall_iterations, min_c
            quality_threshold, task, evalue, dust, num_alignments, soft_masking, perc_identity, mode, max_read_size,
            with_indels, stretches_pvalue, stretches_distance, stretches_to_plot, cleanup,
            cpu_count, overlapping_reads, db_path, max_memory, calculate_haplotypes="Y"):
+    if os.path.exists(output_dir): # the output folder must be empty
```

Comment:
note bellow that output_dir can be None which would result in an error here...
So this check should happen after assigning output_dir (a few lines down)

Reply:
also the comment is redundant